api/src/routers/meals.js

```diff
@@ -0,0 +1,82 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try {
+    const [allMeals] = await knex.raw("SELECT * FROM Meal");
+    res.json(allMeals);
+  } catch (err) {
+    res.status(500).json({ error: err.message });
+  }
+});
+
+router.post("/api/search", async (req, res) => {
+  if (!req.body.title || req.description.length < 3) {
+    return res
+      .status(400)
+      .send("Missing required fields: name, price, description");
+  }
+  try {
+    const newMeal = req.body;
```

Comment:
maybe it would be worth here to add some checks whether the obj coming as body is worthy being inserted into db :)